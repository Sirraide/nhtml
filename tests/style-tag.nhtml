style {
    .flex {
        display: flex;
        flex-direction: row;
    }
}

div.flex {
    .flex (test)
}