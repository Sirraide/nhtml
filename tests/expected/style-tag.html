<style>
    .flex {
        display: flex;
        flex-direction: row;
    }
</style>
<div class='flex'>
    <div class='flex'>test</div>
</div>
